src/entries/popup/hooks/swap/useSwapInputs.ts

```diff
@@ -247,6 +251,97 @@ export const useSwapInputs = ({
     [assetToBuyValue, independentField],
   );
 
+  const BACKEND_CHAINS = useMemo(
+    () => transformBackendNetworksToChains(backendNetworks.networks),
+    [],
+  );
+  const determineOutputCurrency = useCallback(
+    (inputCurrency: ParsedSearchAsset | null) => {
+      if (!inputCurrency) return null;
+
+      const currentChainId = inputCurrency.chainId;
+
+      // Find the current chain in BACKEND_CHAINS
+      const currentChain = BACKEND_CHAINS.find(
+        (chain) => chain.id === currentChainId,
+      );
+
+      if (currentChain && currentChain.nativeCurrency.symbol === 'ETH') {
```

Comment:
there's no `isNative` param to be used here instead of comparing with ETH? thinking about other networks native assets too